TODO.md

```
@@ -30,6 +30,9 @@ These items have dependencies and should be done in order.
 
 ## To plan/design:
 
+### Optional config escape hatch for project-wide orchestrator session listing
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)